### DISCUSSION TITLE
Can't get rid of demo flows

### DISCUSSION THREAD

**Discussion by j2sw** [akvorado/akvorado#1456] [Q&A]
I am a new user.  Went through the docs. I did the following:

Remove the reference for docker-compose-demo.yml from .env,
Comment the last line of akvorado.yaml, and
Run docker compose up -d --remove-orphans.

I still have the demo flows.

**Comment by vincentbernat** [comment]
Are new demo flows still generated or do you mean the previously generated flows are still visible? If the later, this is expected.

**Reply by Tayyab146** [reply]
Seeing same issue, removed teh demo.yml file as well, still seeing dc3, dc5 th2 example exporters exporting data to akvorado

**Reply by vincentbernat** [reply]
What's the output of `docker ps | grep akvorado-exporter`?

**Reply by Tayyab146** [reply]
Output comes out nothing, But I've attached the console UI
![image](https://github.com/user-attachments/assets/cec33211-b886-4492-a7ff-1edc5005c209)

**Reply by Tayyab146** [reply]
@vincentbernat any update?

**Reply by j2sw** [reply]
I get the same behavior at Tayyab146 I get no results when i run the docker ps | grep mentioned above

**Reply by Tayyab146** [reply]
> I get the same behavior at Tayyab146 I get no results when i run the docker ps | grep mentioned above

Delete the docker volumes by disabling the demo data and hopefully it'll resolve the incoming demo data.
